R/test-that.r

```diff
@@ -73,6 +82,28 @@ test_code <- function(description, code, env) {
   invisible(ok)
 }
 
+report_results <- function(results) {
```

Comment:
Then we'll add as.expectation.error(), as.expectation.skip() (as replacement for current expecation_error() and expectation_skip()) and later as.expectation.warning() (#310).